Solutions/Corvus.SpecFlow.Extensions.Specs/Corvus.SpecFlow.Extensions.Specs.csproj

```diff
@@ -19,11 +27,22 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
   </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Corvus.SpecFlow.Extensions\Corvus.SpecFlow.Extensions.csproj" />
+  </ItemGroup>
  
   <ItemGroup>
     <None Update="specflow.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
+ 
+  <ItemGroup>
+    <SpecFlowFeatureFiles Update="Containers\PerScenarioContainer.feature">
+      <Visible>$(UsingMicrosoftNETSdk)</Visible>
+      <CodeBehindFile>%(RelativeDir)%(Filename).feature$(DefaultLanguageSourceExtension)</CodeBehindFile>
+    </SpecFlowFeatureFiles>
+  </ItemGroup>
 
 </Project>
```

Comment:
Shouldn't this reference our stylecop.json to enforce coding standards? I know we don't tend to provide full XML doc comments for step files, but we should still be adhering to our coding standards, file headers, etc.

Reply:
Yes, that was an omission. I've put this in and also addressed all the resulting things it found in an update to this PR.